drivers/clock_control/Kconfig.silabs

```diff
@@ -0,0 +1,11 @@
+# Silicon Labs Series 2+ CMU control driver
+
+# Copyright (c) 2024 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config CLOCK_CONTROL_SILABS_SERIES
+	bool "Silicon Labs Series 2+ Clock Management Unit clock control driver"
```

Comment:
Any idea to get a shorter entry? Use can still acces to the help to get the full description.

Reply:
Dropped "Clock Management Unit" from the title.